.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,100 @@
+---
+name: Test Windows
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        description: >
+          Python version to currently test
+        type: string
+        required: true
+
+      runner:
+        description: >
+          Describes the system this workflow should run on.
+        type: string
+        required: true
+
+      install-optionals:
+        description: >
+          Decides whether we install optyional dependencies
+        type: boolean
+        default: false
+
+      use-msrv:
+        description: >
+          Decides whether we use MSRV or stable for rust toolchain.
+        required: false
+        type: boolean
+        default: false
```

Comment:
Same here remove this option and always use msrv.

Reply:
Removed in 8d54c24